### PR TITLE
Add test launcher for minimal test plan

### PR DESCRIPTION
**Pull request by petitfou**
- Add minimal testplan & test-runner-automated-minimal app
- Add script run-minimal to manage conflict with debian checkbox package & run minimal test
- Add intel-gpu-tools into the provider snap

### CODE REVIEW

**Review by mckees — APPROVED** [review]

**Review by kissiel — COMMENTED** [review]
Some logical inconsistency below

**Comment by mckees** [comment]
All tests pass on my RPL machine on the candidate hwaccel chromium channel

File: launchers/test-runner-automated-minimal [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+forced = yes
+
+[test selection]
+forced = yes
+
+[ui]
+type = silent
+auto_retry = no
+max_attempts = 4
+delay_before_retry = 30
```
[the remark this answers is not in the document]
Comment (petitfou) [reply launchers/test-runner-automated-minimal:18]: ok thanks

File: launchers/test-runner-automated-minimal [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+unit = com.canonical.certification::kivu-minimal
+forced = yes
+
+[test selection]
+forced = yes
+
+[ui]
+type = silent
+auto_retry = no
+max_attempts = 4
```
[the remark this answers is not in the document]
Comment (petitfou) [reply launchers/test-runner-automated-minimal:17]: ok thanks

File: launchers/test-runner-automated-minimal (hunk 1 above)
[the remark this answers is not in the document]
Comment (kissiel) [reply launchers/test-runner-automated-minimal:17]: any reason to leave the delay for retries?

**Pull request merged**